source/collaborate/schedule-messages.rst

```diff
@@ -0,0 +1,43 @@
+Schedule messages
```

Comment:
1. Can we also highlight the fact that scheduled posts can have a post priority, attachments or basically anything that a regular post can contain.

2. Can we also highlight that scheduled posts also have to go through the following hooks, just like a regular post would- `MessageHasBeenPosted` and `MessageWillBePosted` on the server side at the scheduled time of posting, and  hooks registered via the `registerMessageWillBePostedHook` on webapp will be run at the tie of scheduling the post.

Reply:
@harshilsharma63 - Given that this docs PR focuses on end user documentation, this isn't the best place to cover things like hooks. Our dev docs may be a better place. Would you be open to drafting a dev docs update for scheduled messages?